topics/admin/tutorials/upgrading/tutorial.md

```diff
@@ -56,13 +57,103 @@ We can upgrade our Galaxy version by modifying the Ansible var files and re-runn
 
 This tutorial will show you how and discuss some of the things you need to keep in mind whenever you are updating or upgrading your Galaxy server.
 
-## Some notes on Galaxy versioning and releases.
+> <hands-on-title>Upgrade checklist</hands-on-title>
+> If you are ready to perform your upgrade, the following checklist could inspire you into performing some important steps to ensure a successful Galaxy upgrade. It includes points important in a change management process.
+>
+> ## Pre-upgrade
+> - [ ] Review release notes for new features, deprecations, and breaking changes 
+> - [ ] Assess the upgrade's impact on workflows, tools, data formats, and integrations
+> - [ ] Evaluate changes in resource requirements and dependencies:
+>   - Python version 
+>   - libraries/containers
+>   - Linux distribution
+>   - VM size and computational powers
+>   - Others depending on set-up 
```

Comment:
The minimum version of the database server (PostgreSQL) is usually another important one.